exporter/opentelemetry-exporter-jaeger-proto/src/opentelemetry/exporter/jaeger/proto/__init__.py

```diff
@@ -0,0 +1,176 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+
+OpenTelemetry Jaeger Protobuf Exporter
+--------------------------------------
+
+The **OpenTelemetry Jaeger Protobuf Exporter** allows to export `OpenTelemetry`_ traces to `Jaeger`_.
+This exporter always sends traces to the configured agent using Protobuf via gRPC.
+
+Usage
+-----
+
+.. code:: python
+
+    from opentelemetry import trace
+    from opentelemetry.exporter.jaeger.proto import JaegerExporter
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+    trace.set_tracer_provider(TracerProvider())
+    tracer = trace.get_tracer(__name__)
+
+    # create a JaegerExporter
+    jaeger_exporter = JaegerExporter(
+        # optional: configure collector
+        # collector_endpoint='localhost:14250',
+        # insecure=True, # optional
+        # credentials=xxx # optional channel creds
+        # max_tag_value_length=None # optional
+    )
+
+    # Create a BatchSpanProcessor and add the exporter to it
+    span_processor = BatchSpanProcessor(jaeger_exporter)
+
+    # add to the tracer
+    trace.get_tracer_provider().add_span_processor(span_processor)
+
+    with tracer.start_as_current_span('foo'):
+        print('Hello world!')
+
+You can configure the exporter with the following environment variables:
+
+- :envvar:`OTEL_EXPORTER_JAEGER_ENDPOINT`
+- :envvar:`OTEL_EXPORTER_JAEGER_CERTIFICATE`
+
+API
+---
+.. _Jaeger: https://www.jaegertracing.io/
+.. _OpenTelemetry: https://github.com/open-telemetry/opentelemetry-python/
+"""
+# pylint: disable=protected-access
+
+import logging
+from os import environ
+from typing import Optional
+
+from grpc import ChannelCredentials, insecure_channel, secure_channel
+
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger.proto import util
+from opentelemetry.exporter.jaeger.proto.gen import model_pb2
+from opentelemetry.exporter.jaeger.proto.gen.collector_pb2 import (
+    PostSpansRequest,
+)
+from opentelemetry.exporter.jaeger.proto.gen.collector_pb2_grpc import (
+    CollectorServiceStub,
+)
+from opentelemetry.exporter.jaeger.proto.translate import (
+    ProtobufTranslator,
+    Translate,
+)
+from opentelemetry.sdk.environment_variables import (
+    OTEL_EXPORTER_JAEGER_ENDPOINT,
+)
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+
+DEFAULT_GRPC_COLLECTOR_ENDPOINT = "localhost:14250"
+
+logger = logging.getLogger(__name__)
+
+
+class JaegerExporter(SpanExporter):
+    """Jaeger span exporter for OpenTelemetry.
+
+    Args:
+        collector_endpoint: The endpoint of the Jaeger collector that uses
```

Comment:
Not too familiar with Jaeger. When are the other constructor arguments used? `agent_*`  related args to be exact.